static/styles/settings.css

```diff
@@ -882,15 +882,22 @@ input[type="checkbox"] {
     margin-top: 5px;
 }
 
-.edit_bot_form_container {
-    padding: 15px;
+#edit_bot_modal {
+    /* Ensure that the dropdown can overflow the modal. */
+    .modal__content {
+        overflow: visible;
+    }
 }
 
 .edit_bot_form {
     font-size: 100%;
     margin: 0;
     padding: 0;
 
+    form {
+        margin: 0;
+    }
```

Comment:
Is this required, we already set the margin to 0 above and form element itself as `edit_bot_form` class.